include/boost/random/detail/uniform_int_float.hpp

```diff
@@ -60,7 +60,7 @@ class uniform_int_float
     result_type operator()()
     {
         base_result range = static_cast<base_result>((max)())+1;
-        return static_cast<result_type>(_rng() * range);
+        return static_cast<result_type>(static_cast<base_result>(_rng() * range));
```

Comment:
AMDG

On 03/12/2015 12:00 PM, jzmaddock wrote:
>
> I think that's all the points you raised addressed now?
> 

I think so.  I'll try to review it again
by the end of this week.

In Christ,
Steven Watanabe